incubator/cassandra/templates/backup/cronjob.yaml

```diff
@@ -35,7 +35,7 @@ spec:
             - --namespace
             - {{ $release.Namespace }}
             - --selector
-            - release={{ $release.Name }}
+            - app={{ template "cassandra.name" $ }}
```

Comment:
Are you sure this shouldn't be equal to line 12 ?

Reply:
Should be the same as the StatefulSet app label, and different from the CronJobs app label

Reply:
OK. What we have 2 releases in one namespace ? 
Should we leave release={{ $release.Name }} part ?